src/markdown/directives/parse.js

```diff
@@ -20,6 +28,20 @@ import marpitPlugin from '../marpit_plugin'
 function parse(md, opts = {}) {
   const { marpit } = md
 
+  const applyBuiltinDirectives = (newProps, builtinDirectives) => {
+    let ret = {}
+
+    for (const prop of Object.keys(newProps)) {
+      if (builtinDirectives[prop]) {
+        ret = { ...ret, ...builtinDirectives[prop](newProps[prop], marpit) }
```

Comment:
When aliasing from custom directive to built-in directive, we will re-apply function of built-in directives to allow only valid value.